README.md

```diff
@@ -30,53 +62,93 @@ There are some configuration options for the sysadmins out there. All configurat
 * `master` *(array)* Pubkeys of users who, if they connect to the Scuttlebot instance, are allowed to command the primary user with full rights. Useful for remotely operating a pub. Defaults to `[]`.
 * `logging.level` *(string)* How verbose should the logging be. Possible values are error, warning, notice, and info. Defaults to `notice`.
 
-### `connections`
+Deprecated Options:
+* `host` *(string)* The domain or ip address for `sbot`. Defaults to your public ip address.
+* `port` *(string|number)* The port for `sbot`. Defaults to `8008`.
 
-Two objects to specify `incoming` and `outgoing` transports and transformations for connections.
+You should use `connections` to more explicitly configure connections.
 
-The default is the following. It specifies the default TCP `net`work transport for incoming and outging connections, using secret-handshake/boxstream ([shs](https://github.com/auditdrivencrypto/secret-handshake)) for authentication and encryption.
+### `connections`
 
+An object with two required properties: `incoming` and `outgoing` to specify transports and transformations for connections.
+Defaults to the following:
 ```json
-"connections": {
+{
   "incoming": {
     "net": [{ "port": 8008, "scope": "public", "transform": "shs" }]
   },
   "outgoing": {
     "net": [{ "transform": "shs" }]
   }
-},
+}
 ```
 
-If you want to use [Tor](https://torproject.org) to create outgoing connections you can specify your `outgoing` like this. It will use `localhost:9050` as the socks server for creating this.
+It specifies the default TCP `net`work transport for incoming and outging connections, using secret-handshake/boxstream ([shs](https://github.com/auditdrivencrypto/secret-handshake)) for authentication and encryption.
+
+A **transport** is a vehicle or avenue for communication. The following transports are currently supported:
+- `net` - TCP based
+- `unix` - socket based
+- `onion` - TOR based
+- `ws` - websocket based
+
+Each transport can have an array of different configurations passed to it, these are objects with properties:
+- `transform` *(string)* determines whether traffic is encrypted, and if so how.
+  - `shs` - secret handashake
+  - `noauth` - no encryption
+- `port` *(integer)*
+- `host` *(string)* only relevant for ... TODO
+- `scope` *(string)* ... TODO
+  - `private` - TODO
+  - `public` - TODO
+- `external` *(array of strings)* ... TODO
```

Comment:
Thanks for leaving TODOs, this makes it much easier to figure out what needs to be documented in the future.